api/api_sources/sources/database/database-schema/schema-sqls/SlopeCodeSchema.sql

```diff
@@ -0,0 +1,39 @@
+-- ### Creating Table: observation_slope_code ### --
+
+        
+CREATE TABLE observation_slope_code ();
+ALTER TABLE observation_slope_code ADD COLUMN observation_slope_code_id SERIAL PRIMARY KEY;
+ALTER TABLE observation_slope_code ADD COLUMN observation_slope_code VARCHAR(5) NOT NULL UNIQUE;
+ALTER TABLE observation_slope_code ADD COLUMN description VARCHAR(100) NULL;
+ALTER TABLE observation_slope_code ADD COLUMN active_ind BOOLEAN NOT NULL DEFAULT TRUE;
+
+
+        
+-- ### Creating Comments on table ### --
+
+        
+COMMENT ON TABLE observation_slope_code IS 'Observation area slope code. Slope code is integer encoded enum values [0 to 9]. O specified data not available and 1 means flat with 0%, similarly 9 specified very steep slop with more than 45%';
```

Comment:
Should update the table comment to reflect the new codes. Second sentence refers to the original numeric codes.

Reply:
sure I will update description

Reply:
May also want to imply that "slope" reflects the steepness of the ground at the point of observation...